uint8 speed_left
uint8 speed_right
bool direction_left 
bool direction_right
uint8 pitch
uint8 roll

